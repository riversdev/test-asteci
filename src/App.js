import { Table } from './components/Table'

const App = () => {
  return (
    <div className='container'>
      <h1>Test</h1>
      <hr />
      <Table />
    </div>
  )
}

export default App