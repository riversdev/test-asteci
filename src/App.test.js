import { render, screen } from "@testing-library/react"
import App from "./App"

describe('tests App', () => {
    test('should display all elements', () => {
        render(<App />)

        expect(screen.getByText('Test')).toBeTruthy()
        expect(screen.getByText('Cargando...')).toBeTruthy()

        // screen.debug()
    })

    test('should match with snapshot', () => {
        const { container } = render(<App />)

        expect(container).toMatchSnapshot()
    })
})