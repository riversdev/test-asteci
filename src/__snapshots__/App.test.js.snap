// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`tests App should match with snapshot 1`] = `
<div>
  <div
    class="container"
  >
    <h1>
      Test
    </h1>
    <hr />
    <p>
      Cargando...
    </p>
  </div>
</div>
`;
